### ISSUE TITLE
CSS Colour Contrasts

### ISSUE THREAD

**Issue by ameyannamma** [ameyannamma/jk-rowling-fan-page#8]
CSS - Background colours should be combined with legible text confirming to WCAG 2.0 AA standards

**Comment by abitsalihu** [comment]
Hey @ameyannamma, I would like to help on this issue.
